iiblib/iib_build_details_model.py

```diff
@@ -536,7 +536,7 @@ class AddDeprecationsModel(IIBBuildDetailsModel):
         binary_image_resolved (str)
             A checksum reference of binary image that was used for rebuilding
         deprecation_schema (str)
-            A link to another endpoint that provides the deprecation schema for the request
+            A JSON string of deprecation schema
```

Comment:
Sorry for the confusion, I thought this will indicate the input, but now I get that this is the output of build details.
deprecation_schema will be inputed as json string but in build details it will be shown as a link to deprecation schema

Reply:
The attribute in build details is changed to deprecation_schema_url.